### PR TITLE
Modified README.md file according to requirements

### LINKED ISSUE #26: Show support
Append the following to `README.md`.

```
### Show your support
Give a ⭐️ if this project helped you!

### License
This project is [MIT](https://github.com/khattakdev/capture/blob/main/LICENSE.md) licensed.
```

### PR DESCRIPTION
**Pull request by Kola-Prabhas**
I have modified the README.md file and added the required information.
I noticed that the path of preview.gif is missing a period(.) and updated it and now it shows the gif.
please provide additional feedback if any changes are necessary.
closes #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Capture
 
 A VS Code plugin that generates a screenshot of the highlighted content.
-![demo - how to capture](/assets/preview.gif)
+![demo - how to capture](./assets/preview.gif)
```
Comment (khattakdev) [inline README.md:4]: This will workout without dot as well, making it an absolute path.

**Comment by Kola-Prabhas** [comment]
@khattakdev  Should I remove the dot.

**Comment by khattakdev** [comment]
Yes

**Comment by Kola-Prabhas** [comment]
@khattakdev I've removed the dot from the path.
please provide information ,if any changes needed

**Pull request merged**